External/AvalonTools/CMakeLists.txt

```diff
@@ -36,6 +36,16 @@ if(needDownload)
     WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR})
 endif()
 
+set(reaccsio_c "${AVALONTOOLS_DIR}/src/main/C/common/reaccsio.c")
```

Comment:
Is this really a better way of solving this than using a traditional patch?

Reply:
@greglandrum No, it isn't. But it is cross-platform as it is supported by CMake, while `patch` is not available on Windows and it is not supported by CMake. I also hope this is only a temporary solution until a new version of Avalon becomes available upstream.